2-mandatory/3-stocks.js

```diff
@@ -33,22 +34,39 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Solve the smaller problems, and then build those solutions back up to solve the larger problem.
         Functions can help with this!
 */
+
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+  let averageValue = [];
+  for (const i of closingPricesForAllStocks) {
+    let sum = 0;
+    let average = 0;
+    for (const j of i) {
```

Comment:
The implementation is good.
But it's worth keeping an eye on variable names. It's usually a good idea to have more meaningful names, so it is easier for other developers to understand.